model/preference.go

```diff
@@ -33,7 +33,9 @@ const (
 
 	PREFERENCE_CATEGORY_NOTIFICATIONS = "notifications"
 	PREFERENCE_NAME_EMAIL_INTERVAL    = "email_interval"
-	PREFERENCE_DEFAULT_EMAIL_INTERVAL = "30" // default to match the interval of the "immediate" setting (ie 30 seconds)
+
+	PREFERENCE_EMAIL_INTERVAL_IMMEDIATE       = "30"  // the "immediate" setting is actually 30s
+	PREFERENCE_EMAIL_INTERVAL_FIFTEEN_MINUTES = "900" // fifteen minutes is 900 seconds
```

Comment:
Ideally, these default values would be shared between server and client, but I have already burned a lot of time on this ticket. Maybe an improvement for the future?